features/admin.identity-providers.v1/components/forms/authenticators/push-authenticator-form.tsx

```diff
@@ -0,0 +1,520 @@
+/**
```

Comment:
Do we need to add this to admin.identity-providers.v1?

Reply:
We still use AuthenticatorFormFactory from identity-providers feature to render the authenticator's edit form. Therefore we still need to add these to identity providers module

[1] https://github.com/wso2/identity-apps/blob/master/features/admin.connections.v1/components/edit/edit-multi-factor-authenticator.tsx#L24